src/sparse_set/mod.rs

```diff
@@ -104,6 +104,35 @@ impl<T: Component> SparseSet<T> {
     pub fn as_slice(&self) -> &[T] {
         &self.data
     }
+    /// Displays storages memory information.
+    pub fn memory_usage(&self) -> Option<StorageMemoryUsage> {
+        Some(StorageMemoryUsage {
+            storage_name: type_name::<Self>().into(),
+            allocated_memory_bytes: self.allocated_memory_bytes(),
+            used_memory_bytes: self.used_memory_bytes(),
+            component_count: self.len(),
+        })
```

Comment:
It doesn't need the `Option` here.

```suggestion
    pub fn memory_usage(&self) -> StorageMemoryUsage {
        StorageMemoryUsage {
            storage_name: type_name::<Self>().into(),
            allocated_memory_bytes: self.allocated_memory_bytes(),
            used_memory_bytes: self.used_memory_bytes(),
            component_count: self.len(),
        }
```